package-lock.json

```diff
@@ -19834,6 +19834,93 @@
 						"worker-rpc": "^0.1.0"
 					}
 				},
+				"fork-ts-checker-webpack-plugin-v5": {
```

Comment:
I'll bet you $10 my next `npm install` will put this `fork-ts-checker-webpack-plugin-v5` somewhere else in this file 😂

Reply:
Do you accept € as well? There is no way I will win against that bet 😂